R/convGDX2MIF.R

```diff
@@ -115,7 +115,17 @@ convGDX2MIF <- function(gdx, gdx_ref = NULL, file = NULL, scenario = "default",
   output <- add_dimension(output,dim=3.1,add = "model",nm = "REMIND")
   output <- add_dimension(output,dim=3.1,add = "scenario",nm = scenario)
 
+  message("Checking integrity of created MIF...")
```

Comment:
Not sure about this message.  This is checking only one aspect of data integrity.  And since we expect the reporting to be error-free, it would only add a meaningless line to the log file

Reply:
Happy to delete it :)